presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -77,6 +82,29 @@ public void testSTPolygon()
         assertInvalidFunction("ST_AsText(ST_Polygon('LINESTRING (1 1, 2 2, 1 3)'))", "ST_Polygon only applies to POLYGON. Input type is: LINE_STRING");
     }
 
+    @Test
+    public void testSTGeometryType()
+    {
+        Map<GeometrySerializationType, String[]> types = new ImmutableMap.Builder<GeometrySerializationType, String[]>()
+                .put(GeometrySerializationType.POINT, new String[]{"ST_GeometryType(ST_Point(1, 4))", "ST_Point"})
+                .put(GeometrySerializationType.LINE_STRING, new String[]{"ST_GeometryType(ST_GeometryFromText('LINESTRING (1 1, 2 2)'))", "ST_LineString"})
+                .put(GeometrySerializationType.POLYGON, new String[]{"ST_GeometryType(ST_GeometryFromText('POLYGON ((1 1, 1 4, 4 4, 4 1))'))", "ST_Polygon"})
+                .put(GeometrySerializationType.MULTI_POINT, new String[]{"ST_GeometryType(ST_GeometryFromText('MULTIPOINT (1 1, 2 2)'))", "ST_MultiPoint"})
+                .put(GeometrySerializationType.MULTI_LINE_STRING, new String[]{"ST_GeometryType(ST_GeometryFromText('MULTILINESTRING ((1 1, 2 2), (3 3, 4 4))'))", "ST_MultiLineString"})
+                .put(GeometrySerializationType.MULTI_POLYGON, new String[]{"ST_GeometryType(ST_GeometryFromText('MULTIPOLYGON (((1 1, 1 4, 4 4, 4 1)), ((1 1, 1 4, 4 4, 4 1)))'))", "ST_MultiPolygon"})
+                .put(GeometrySerializationType.GEOMETRY_COLLECTION, new String[]{"ST_GeometryType(ST_GeometryFromText('GEOMETRYCOLLECTION(POINT(4 6),LINESTRING(4 6,7 10))'))", "ST_GeomCollection"})
+                .put(GeometrySerializationType.ENVELOPE, new String[]{"ST_GeometryType(ST_Envelope(ST_GeometryFromText('LINESTRING (1 1, 2 2)')))", "ST_Polygon"})
+                .build();
+
+        for (GeometrySerializationType t : GeometrySerializationType.values()) {
```

Comment:
@jwass The original version of the test was more readable and with `standardName` getter in the `GeometryType` enum this is no longer necessary.